src/server/oasisapi/portfolios/models.py

```diff
@@ -55,13 +90,27 @@ def __str__(self):
 def delete_connected_files(sender, instance, **kwargs):
     """ Post delete handler to clear out any dangaling analyses files
     """
-    files_for_removal = [ 
+    files_for_removal = [
          'accounts_file',
          'location_file',
          'reinsurance_info_file',
          'reinsurance_scope_file',
-    ]   
+    ]
     for ref in files_for_removal:
         file_ref = getattr(instance, ref)
         if file_ref:
             file_ref.delete()
+
+
+def set_portolio_valid(instance):
+    oed_files = [
```

Comment:
missing docstring

Reply:
Moved it to an internal method rather than a stand alone function, which makes more sense